docs/samples/gcp-iap/README.md

```diff
@@ -0,0 +1,137 @@
+## KFServing GCP/IAP Example 
+When using Kubeflow with GCP it is common to use a [GCP Identity Aware Proxy](https://cloud.google.com/iap) (IAP) to manage client authentication to the KFServing endpoints.  The proxy intercepts and authenticates users and passes identity assertion (JWT) to kubernetes service/pods.  Whilst it is also possible to add access control (i.e. programmable or service mesh authorization), this is not described here.
+
+### Prerequisites
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
```

Comment:
might want to mention the kubeflow installation as well
* You can use [kfctl_gcp_iap.v1.0.2.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_gcp_iap.v1.0.2.yaml), editing the repos section in the yaml before deploying to your Kubernetes cluster which contains [knative istio probe fix](https://github.com/kubeflow/manifests/commit/928cf483361730121ac18bc4d0e7a9c129f15ee2).

    ```diff
        repos:
        - name: manifests
    +     uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
    -     uri: https://github.com/kubeflow/manifests/archive/v1.0.2.tar.gz
    -   version: v1.0.2
    ```